.github/workflows/build.yml

```diff
@@ -109,6 +109,8 @@ jobs:
       - name: Pre-docs-Publish
         run: |
           yarn docs
+        env:
+          NODE_OPTIONS: --max_old_space_size=4096
```

Comment:
should we merge-back together steps Pre-npm-Publish and Pre-docs-Publish? Also step max_old_space_size to 8192 (4096 failed locally for me)

Reply:
I was afraid of being too much, but that's only an upper bound and it will use as "little" as it can anyway. Will do.

Reply:
I'll keep both separated as it "isolates" the CI secrets a bit more...